src/components/data_comps_nuopc/datm/src/atm_comp_nuopc.F90

```diff
@@ -427,6 +428,10 @@ subroutine InitializeRealize(gcomp, importState, exportState, clock, rc)
     if (ChkErr(rc,__LINE__,u_FILE_u)) return
     call t_stopf('datm_strdata_init')
 
+    ! Write mesh for debugging
```

Comment:
Do we need this?

Reply:
It was just for debugging, I think I forgot to remove. I have just removed it.